Requested changes before going online

deinhofer opened the pull request:
Please check the comments, which I will add later and update the branch accordingly.

There are still some technical issues and also content-related issues.

Commit:
changed formatting of script tags

deinhofer commented on custom/bin/ARE/web/webapps/asterics-smarthome-knx/index.html:
```diff
@@ -26,41 +26,41 @@
+	
+			var modelFilePathOnWebserver='webapps/asterics-smarthome-knx/models/KNX.acs';
+	
+			function applyAndStart() {
```
remove applyAndStart, use applySettingsInXMLModel for button "Start Model" instead.

Commit:
Renamed start.html to index.html so that the demo is not opened in the iframe but uses the whole page

Commit:
The following changes were done:
 * changed some formatings
 * use applySettingsInXMLModelAndStart instead of applyAndStart
 * Changed some texts
 * moved descriptive images to "More Info" section because this Info is just needed if you want to understand whats happening under the hood.
 * removed copied text of KNX-plugin help, added link to help instead.

TODO for @stefanstocki
 * Please mark the respective areas of the image: What is heating, cooling, lighting, ...
 * Please add a link to the system architecture schematic file and the group addresses export from the configuration SW
 * Check script loading errors of popper.js,...

deinhofer commented:
@stefanstocki 
Please commit your changes to this branch:
```
git pull
git checkout fixesForIndexHTMLAndRepo
```
So that I can review them finally.

Commit:
There was a confusion regarding folder names:

The Asterics framework has the folders

web/webapps/startpage/styles
web/webapps/startpage/lib

which contains global styles and javascript libs for asterics webapps.

This repo had a folder
web/webapps/startpage/style
web/webapps/startpage/lib

As those styles and libs are solution specific and should not overwrite
the global styles and libs they were moved to the

web/webapps/asterics-smarthome-knx/styles
web/webapps/asterics-smarthome-knx/lib

folders. Besides style was renamed to styles to have a common naming.
Some unused .css and .js files were removed

The script jquery-3.2.1.js was removed and
../startpage/clientExample/javascript/jquery-3.2.1.min.js was used.
The libs in clientExample/javascript should one day moved to
web/webapps/startpage/lib as well.

deinhofer commented:
@stefanstocki  I already fixed the .css and .js loading errors and cleaned up unused files.
These 2 todos are still left for you:
* Please mark the respective areas of the image: What is heating, cooling, lighting, ...
 * Please add a link to the system architecture schematic file and the group addresses export from the configuration SW

ghost commented:
you wrote [here](https://es.technikum-wien.at/StudyATHome/StudyATHome/issues/2): 

> Schließen wir die Demo so ab wie sie ist, den Rest schauen wir uns später an.

what should i do now? add knx basics and demo kit architecture, or just the ETS group address Export, as discussed in the meeting a few weeks before?

deinhofer commented:
@stefanstocki 
as just discussed: No KNX basics, but provide topological figure of demo kit architecture and ETS group address export (with english texts). So that somebody could imagine how to build and setup the HW on his own and then use the AsTeRICS KNX demo.

Commit:
added knx setup overview and groudaddress export

Commit:
changed pics

Commit:
moved <hr> line to bottom of 'Start Model' section
Fixed saveSettingsAsAutostartModel to saveSettingsAsAutostartModel(modelFilePathOnWebserver)

deinhofer committed:
added KNX system and group addresses to requirements list

deinhofer commented:
All requested changes done.
demo kit picture looks good now
